CONTRIBUTING.md

```diff
@@ -31,12 +31,53 @@ Any module must follow this basic format:
 
 ```js
 module.exports = function ModuleName(options,UI) {
+
   options = options || {};
   options.title = "Title of the Module";
   UI.onSetup(options.step);
   var output;
 
   function draw(input,callback) {
+
+    UI.onDraw(options.step);
+
+    var output = /*do something with the input*/ ;
+
+    this.output = output;
+    options.step.output = output.src;
+    callback();
+    UI.onComplete(options.step);
+  }
+
+  return {
+    options: options,
+    draw: draw,
+    output: output,
+    UI: UI
+  }
+}
+```
+The default loading spinner can be overriden with a custom progress object to draw progress on the CLI, following is a basic module format for the same
```

Comment:
i know it's pretty clear, but let's just say `can be optionally overridden` -- how does that sound?

Reply:
@jywarren that sounds awesome😁